soccer/src/soccer/planning/planner/escape_obstacles_path_planner.hpp

```diff
@@ -17,10 +17,16 @@ namespace planning {
  * @brief This planner finds a path to quickly get out of an obstacle. If the
  * start point isn't in an obstacle, returns a path containing only the start
  * point.
+ *
+ * Params taken from MotionCommand:
+ *   None
  */
-class EscapeObstaclesPathPlanner : public Planner {
+class EscapeObstaclesPathPlanner : public PathPlanner {
 public:
-    EscapeObstaclesPathPlanner() : Planner("EscapeObstaclesPathPlanner"){};
+    // TODO(Kevin): think of better way to convey halted behavior than "EscapeObstaclesPathPlanner"
+    // actually this is not HALT, control node does HALT for us
+    // or maybe use PathTargetPlanner with no real velocity?
```

Comment:
HALT is actually implemented control-side, the control node simply gives every HALT robot 0 velocity. It's annoying because we have to give some non-empty Trajectory for that to happen though (hence why EscapeObstacles is the default)

Reply:
https://github.com/RoboJackets/robocup-software/pull/2020/files#diff-0dc67989d0c9e16403a6d3b3bb626886e3a97dc0d8eb1460aeb9f5938ec22065R361
related